rukundoeric/Politico
# #171688050: Set up database

#### What does this PR do?
- Setup database
#### Description of Task to be completed?
- Setup database
#### How should this be manually tested?
* clone this Repo https://github.com/rukundoeric/Politico.git
* Run `cd Politico` to navigation to project directory 
* make sure you have `postgres or psql` installed and then Run `./setup.sh` on Windows or `./setup` on Mac to setup the project
* Final, run `yarn run dev` to start the server
* And then Test with Postman
#### What are the relevant pivotal tracker stories?
- [#171688050](https://www.pivotaltracker.com/story/show/171688050)
